index.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0 minimum-scale=1.0">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600&display=swap' rel='stylesheet' />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Portfolio</title>
+
+</head>
+
+<body>
+
+    <header>
+        <div class="appbar-compact">
+            <div class="toolbar">
+                <div class="logo-placeholder">
+                    <a class="navbar-brand" href="#">My Logo</a>
+                </div>
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </div>
```

Comment:
- [ ] Please kindly use the `nav` tag instead of the `div` tag I think the reviewer who reviewed your previous project he/she missed this point so kindly fix this also. :+1: